### ISSUE TITLE
mvp - alt text fixes

### ISSUE THREAD

**Issue by joelanman** [alphagov/datagovuk_find#1647]
- [ ] Home – logo image should have ‘Home’ alt text 

- [ ] Collection page – badge should have empty alt 

- [ ] Collection page – headline statistics – up and down arrows need alt text 

- [ ] Roadmap – egg needs alt (if we keep it)
